scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -889,3 +889,80 @@ def test_slerp_call_time_out_of_range():
         s([0, 1, 2])
     with pytest.raises(ValueError, match="times must be within the range"):
         s([1, 2, 6])
+
+
+def test_spline_trivial():
```

Comment:
Add non-trivial tests.

Try to find a way to somehow check the continuity of the solution.

Reply:
How about calculating angular velocity (and angular acceleration?) close to the break points using a finite difference approximation?

Reply:
So I implemented this method and it looks like there are some bugs which still need to be ironed out. The angular velocities at `key_times + delta_t` agree to within `1e-3`, but the velocities at `key_times - delta_t` do not. I'll keep looking into it.

Reply:
It might be not exactly a bug. What I suggest to do first is to implement angular velocity and acceleration computations (suggested in my email) and plot them as the function of time. It will be clear whether the behavior is correct. If it is correct, you will find a way to check it formally in tests.